google/cloud/bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -242,6 +285,231 @@ void DeleteCluster(google::cloud::bigtable::InstanceAdmin instance_admin,
+  for (auto const& instance : instances) {
+    std::cout << instance.name() << std::endl;
+  }
+
+  std::cout << "\nGet Instance: " << std::endl;
+  auto instance = instance_admin.GetInstance(instance_id.get());
+  std::string instance_detail;
+  google::protobuf::TextFormat::PrintToString(instance, &instance_detail);
+  std::cout << "GetInstance details : " << std::endl << instance_detail;
+
+  std::cout << "\nListing Clusters: " << std::endl;
+  auto cluster_list = instance_admin.ListClusters(instance_id.get());
+  std::cout << "Cluster Name List: " << std::endl;
+  for (auto const& cluster : cluster_list) {
+    std::cout << "Cluster Name: " << cluster.name() << std::endl;
+  }
+
+  std::cout << "\nDeleting Instance: ";
+  instance_admin.DeleteInstance(instance_id.get());
+  std::cout << " Done" << std::endl;
+}
+//! [run instance operations]
+
+// This full example demonstrate various instance operations
+// by initially creating production instance
+void RunFullExample1(std::string project_id, int argc, char* argv[]) {
+  if (argc != 6) {
+    throw Usage{
+        "run-full-example-1: <project-id> <instance-id> <cluster-id-1> "
+        "<zone-1> <cluster-id-2> <zone-2>"};
+  }
+  google::cloud::bigtable::InstanceId instance_id(ConsumeArg(argc, argv));
+  google::cloud::bigtable::ClusterId cluster_id_1(ConsumeArg(argc, argv));
+  std::string const zone_1 = ConsumeArg(argc, argv);
+  google::cloud::bigtable::ClusterId cluster_id_2(ConsumeArg(argc, argv));
+  std::string const zone_2 = ConsumeArg(argc, argv);
+
+  // [START connect_instance_admin]
+  google::cloud::bigtable::InstanceAdmin instance_admin(
+      google::cloud::bigtable::CreateDefaultInstanceAdminClient(
+          project_id, google::cloud::bigtable::ClientOptions()));
+  // [END connect_instance_admin]
+
+  // [START bigtable_check_instance_exists]
+  std::cout << "\nCheck Instance exists: " << std::endl;
+  auto instances = instance_admin.ListInstances();
+  auto instance_name =
+      instance_admin.project_name() + "/instances/" + instance_id.get();
+  bool instanceExists =
+      instances.end() !=
+      std::find_if(
+          instances.begin(), instances.end(),
+          [&instance_name](google::bigtable::admin::v2::Instance const& i) {
+            return i.name() == instance_name;
+          });
+  // [END bigtable_check_instance_exists]
+
+  // Create instance if does not exists
+  if (not instanceExists) {
+    // [START bigtable_create_prod_instance]
+    std::cout << "\nCreating a PRODUCTION Instance: ";
+    google::cloud::bigtable::DisplayName display_name("Put description here");
+
+    // production instance needs at least 3 nodes
+    auto cluster_config = google::cloud::bigtable::ClusterConfig(
+        zone_1, 3, google::cloud::bigtable::ClusterConfig::HDD);
+    google::cloud::bigtable::InstanceConfig config(
+        google::cloud::bigtable::InstanceId(instance_id), display_name,
+        {{cluster_id_1.get(), cluster_config}});
```

Comment:
Sine this snippet would show up as a single unit, would it be possible to set `cluster_id_1` here instead of the top of the file?